app/admin/companies.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Company do
+  # See permitted parameters documentation:
```

Comment:
Do you need these comments?

Reply:
What comment to you mean? The documentation or the frozen_string_literal? If it's the second rubocop forces me to have that. If it's the documentation I can remove it

Reply:
The documentation. I understand it's auto generated, but if you don't need it, remove it.